src/ccmain/tesseractclass.cpp

```diff
@@ -66,6 +66,8 @@ Tesseract::Tesseract()
       BOOL_MEMBER(tessedit_dump_pageseg_images, false,
                   "Dump intermediate images made during page segmentation",
                   this->params()),
+      BOOL_MEMBER(tessedit_do_invert, true,
```

Comment:
Would the shorter form `tessedit_invert` or a more descriptive name `tessedit_auto_invert` be better?